### DISCUSSION TITLE
Language Server hangs with 'Java: Activating...'

### DISCUSSION THREAD

**Discussion by realmilsky** [redhat-developer/vscode-java#3666] [General]
I'm setting up VSCode in a new (corporate) environment and Language Server extension hangs on startup. The log shows the following:

![image](https://github.com/redhat-developer/vscode-java/assets/104407216/81e69536-af77-4a2f-89b0-a17b1288a2dc)

Can someone please explain what's happening here? Writing to the workspace folder doesn't seem to be the issue but I'm guessing that whatever the extension is piping too here is not allowing it to happen.

I'm on:
VSCode version 1.81.1
Language Support for Java version v1.31.0

**Comment by fbricon** [comment]
Something's wrong with the named pipe.

Until this is fixed, try setting `"java.transport" : "stdio"` in your vscode settings. See https://github.com/redhat-developer/vscode-java/issues/3634#issuecomment-2096242414 for more details

**Reply by realmilsky** [reply]
That did it. Thank you sir!

**Comment by realmilsky** [comment]
Solution provided by @fbricon
